publication-commons/src/main/java/no/unit/nva/publication/model/business/TicketStatus.java

```diff
@@ -17,6 +17,7 @@
 public enum TicketStatus {
     PENDING(PENDING_STATUS, "REQUESTED"),
     COMPLETED(COMPLETED_STATUS, "APPROVED"),
+    NOT_APPLICABLE(TicketStatusConstants.NOT_APPLICABLE, "NOT APPLICABLE"),
     CLOSED(CLOSED_STATUS, "REJECTED"),
     REMOVED(REMOVED_STATUS, "REMOVED");
```

Comment:
To be honest, using Constants here in a "Constant" definition, doesn't improve anything, I do it my self, but I don't like it.
Here I had to load the project into my editor to see why you had `NOT_APPLICABLE`, `NOT_APPLICABLE `& `NOT APPLICABLE`

The enum is fine, reading the actual values it all made sense to me, but it took me a couple of minutes... to figure it out.